src/server/services/scripts/generate-data.js

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env node
+
+const args = process.argv.slice(2)
+
+if (args.length === 0) {
+  require('./../generator').generate()
+} else if (args.length === 2) {
+
+  if ( args[0] === '--only') {
+
+    switch (args[1]) {
+      case 'users':
+        require('./../scraper/users').scrape()
```

Comment:
The generator file has three methods: `generate`, `generateUsers` and `generateRepos`. We don't want to include operations from the scraper.

Please, require the `./../generator` file before the `args` declaration and use the aforementioned methods.